src/BasicRum/Beacon/Catcher/Storage/File.php

```diff
@@ -63,6 +63,7 @@ public function generateBundleFromRawBeacons(): int
                 'beacon_data' => file_get_contents($filePath),
             ];
         }
+        $this->deleteRawBeacons($beaconFiles);
```

Comment:
@korzol 

Could you please add one more new line after the "}" ?

Reply:
Added.
Tried to find out if php-cs-fixer can deal with this kind of formatting and found that it's not. I will keep an eye out for this kind of things